src/jmclient/wallet_rpc.py

```diff
@@ -780,15 +781,24 @@ def directsend(self, request, walletname):
             # all error conditions).
             if not self.coinjoin_state == CJ_NOT_RUNNING:
                 raise ActionNotAllowed()
+
+            old_txfee = jm_single().config.get("POLICY", "tx_fees")
+            if "txfee" in payment_info_json:
```

Comment:
The use of `if` here matches the fact that it's not a `required` field in `DirectSendRequest`, so that's correct afaik.